packages/ods/src/components/pagination/src/components/ods-pagination/ods-pagination.scss

```diff
@@ -0,0 +1,69 @@
+:host(.ods-pagination) {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+
+  li {
```

Comment:
try to avoid generic selector + the style is duplicated on `__results` and `__list`